lectures/lecture09/README.md

```diff
@@ -1 +1,173 @@
 # Lecture 9
+
+     _______     _______.  ______     ___      .______    _______
+    |   ____|   /       | /      |   /   \     |   _  \  |   ____|
+    |  |__     |   (----`|  ,----'  /  ^  \    |  |_)  | |  |__
+    |   __|     \   \    |  |      /  /_\  \   |   ___/  |   __|
+    |  |____.----)   |   |  `----./  _____  \  |  |      |  |____
+    |_______|_______/     \______/__/     \__\ | _|      |_______|
+
+    .___________. __    __   _______
+    |           ||  |  |  | |   ____|
+    `---|  |----`|  |__|  | |  |__
+        |  |     |   __   | |   __|
+        |  |     |  |  |  | |  |____
+        |__|     |__|  |__| |_______|
+
+         _______. __    __   _______  __       __
+        /       ||  |  |  | |   ____||  |     |  |
+       |   (----`|  |__|  | |  |__   |  |     |  |
+        \   \    |   __   | |   __|  |  |     |  |
+    .----)   |   |  |  |  | |  |____ |  `----.|  `----.
+    |_______/    |__|  |__| |_______||_______||_______|
+
+
+This is an interactive adventure to teach you about navigating the shell.
+For reference you might be interested in the excellent [course material put out by Software Carpentry](https://swcarpentry.github.io/shell-novice/).
+We'll start with a little introductory material.
+
+## Starting out
+
+To get started, simply type
+
+    ls
+
+and hit return.
+
+This shows the collection of files that are in your current directory (a.k.a. folder).
+The name `ls` is an abbreviation for "list".
+
+This is the simplest type of shell command invocation.
+
+Next just mash the keyboard making a random command, like this:
+
+    asdfasfkjaeasdkjaadsfkljaeflkwae
+
+What happens?
+
+
+## Anatomy of a shell command
+
+More generally, shell commands look like
+
+    command flags arguments
+
+where `flags` and `arguments` are optional.
+We separate these components with spaces (extra spaces are fine).
+
+
+## Arguments
+
+Try this command with an argument:
+
+    ls /
+
+Here we are passing `/` to the `ls` command as an argument to tell it which directory should be listed.
+Specifically, we are telling it to list the very "bottom" directory of the file system, called root.
+We will discuss directory structure later.
+
+Now try
+
+    ls asdfasdf
+
+which should give an error unless you happen to have a file named `asdfasdf` in your current directory.
+Errors are OK.
+They don't cause any harm, and are a natural part of working in the shell.
+
+
+## Flags
+
+Try this command with a flag:
+
+    ls -1 /
+
+Here `-1` is a flag which specifies that the listing should happen in a single column.
+Flags generally go before the arguments, but some commands will allow them in any order.
+
+Does `ls` allow flags after arguments?
+Try it out!
+
+Flags modify some form of the command behavior, and can range from simple to rather complex.
+
+Flags can have arguments, like
+
+    ls -w 15 /
+
+in which we give the `-w` flag the value of 15.
```

Comment:
This isn't what -w does on the native installation of bash on Mac; 15 is being interpreted as a file/directory. I'm coming up short trying to find another flag/argument pairing for use in `ls`